lib/diagnostic.rb

```diff
@@ -6,35 +6,42 @@ class Diagnostic
   # a Fixnum.
   def longest(list)
     # Your code goes here
+    list.max
```

Comment:
Started off right!

What you have to do is to compare each element of the array with another. 

SO you have to iterate through and use the shaceship comparison to check the values of things

` list.max { |a, b| a.length <=> b.length }